frontend/src/components/OrganizationList/OrganizationList.tsx

```diff
@@ -6,15 +6,27 @@ import { Add } from '@material-ui/icons';
 import { OrganizationForm } from 'components/OrganizationForm';
 import { useAuthContext } from 'context';
 
+const GLOBAL_ADMIN = 2;
+const STANDARD_USER = 1;
+
 export const OrganizationList: React.FC<{
   parent?: Organization;
 }> = ({ parent }) => {
-  const { apiPost, apiGet, setFeedbackMessage } = useAuthContext();
+  const { apiPost, apiGet, setFeedbackMessage, user } = useAuthContext();
   const [organizations, setOrganizations] = useState<Organization[]>([]);
   const [dialogOpen, setDialogOpen] = useState(false);
   const history = useHistory();
   const classes = useStyles();
 
+  let userLevel = 0;
```

Comment:
Seems like this is the same code that exists in `Header.tsx`. Maybe we could refactor this code and create a `getUserLevel` function that is used in both files?

Reply:
Ya good idea.

Reply:
Note: let's just use `user.userType` here and check if it's equal to a global admin